ast/parser_ext.go

```diff
@@ -417,18 +417,26 @@ func ParseImports(input string) ([]*Import, error) {
 // ParseModule returns a parsed Module object.
 // For details on Module objects and their fields, see policy.go.
 // Empty input will return nil, nil.
-func ParseModule(filename, input string) (*Module, error) {
-	stmts, comments, err := ParseStatements(filename, input)
+func ParseModule(filename, input string, popts ...ParserOptions) (*Module, error) {
```

Comment:
Not backwards compatible in the strict sense. Callers are probably OK, though. (I think I've commented this on some other commit already... 🔍 )

Reply:
As per that other comment, I've separated this change into its own new exported function with the idea of not breaking any user code.